internal/pkg/cli/cli.go

```diff
@@ -130,3 +132,18 @@ func logRecommendedActions(actions []string) {
 		log.Infof("%s %s\n", prefix, followup)
 	}
 }
+
+func logFriendlyTextIfRegionIsMissing(err error) {
+	if !isMissingRegionErrorInChain(err) {
+		return
+	}
+	log.Infof(`It looks like your AWS region configuration is missing.
```

Comment:
So friendly! 😄 
Do we want to mention `aws configure` as an option, too, or too much?

Reply:
I sorta feel like a user at this point would have already been aware of `aws configure`? And using `aws configure` is the same as the first suggestion (modifying "~/.aws/config" file), I feel like it's probably more straightforward + transparent to tell them which specific file to modify directly than telling them to run a command that eventually modifies the file 🤔  Just my thoughts!